go.sum

```diff
@@ -583,6 +587,7 @@ github.com/imdario/mergo v0.3.12 h1:b6R2BslTbIEToALKP7LxUvijTsNI9TAe80pLWN2g/HU=
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/inconshreveable/mousetrap v1.0.1 h1:U3uMjPSQEBMNp1lFxmllqCPM6P5u/Xq7Pgzkat/bFNc=
 github.com/inconshreveable/mousetrap v1.0.1/go.mod h1:vpF70FUmC8bwa3OWnCshd2FqLfsEA9PFc4w1p2J65bw=
+github.com/influxdata/influxdb v1.7.6/go.mod h1:qZna6X/4elxqT3yI9iZYdZrWWdeFOOprn86kgg4+IzY=
```

Comment:
(😨) Checking why is it adding this big dependency.

Reply:
It's because of their demo application. I guess we can just add an `exclude` line to go.mod. Maybe also send a PR to them to add a separate go.mod for the demo app?

Reply:
Yup, I have already [created a PR for this](https://github.com/axiomhq/hyperloglog/pull/33).

Reply:
Added an explicit `exclude` for the influxdata dependency until the PR is released.